gcp/gcp.go

```diff
@@ -718,8 +742,13 @@ func (gcp *GoogleCloudPrint) assembleJob(job *Job) (*cdd.CloudJobTicket, string,
 
 	gcp.downloadSemaphore.Acquire()
 	t := time.Now()
+	downloadUrl := job.FileURL
```

Comment:
Is this supposed to be behind the flag? It looks this it contradicts with the CL description 'The changes are NoOp unless new flag is passed'...

Reply:
this is a bug fix required for both xmpp and fcm so not behind flag. 

When connected to local GCP server the download url does not have schema information so print jobs fail.

Reply:
thanks!